src/test/common/page/confirm-updated.js

```diff
@@ -0,0 +1,15 @@
+'use strict'
+
+const Confirm = require('./confirm')
+const CONFIRM_UPDATED_PAGE_TITLE = 'GOV.UK - Application updated'
+
+/**
+ * Page object for the confirmation page after submitting the claim.
```

Comment:
Might be worth mention that this is for the update flow, but its the same page as the normal confirm flow.

Reply:
Done.